docs/general/dql/geo.rst

```diff
@@ -244,11 +244,11 @@ into your geographic data::
 
     cr> SELECT distance(capital_location, 'POINT(0.0 90.0)') as from_northpole
     ... FROM country ORDER BY country_code;
-    +--------------------+
-    |     from_northpole |
-    +--------------------+
-    | 4646930.6709925765 |
-    +--------------------+
+    +-------------------+
+    |    from_northpole |
+    +-------------------+
+    | 4646930.675034644 |
```

Comment:
just curious: what caused this to change?

Reply:
The fetch operation does an implicit re-write to source lookup. Now the value is retrieved from the doc-values and they have slightly different precision.

(I think that source vs. non-source between query-then-fetch and query-and-fetch is something we could look into in more detail generally, but later)